fuzz/Cargo.toml

```diff
@@ -0,0 +1,51 @@
+[package]
+name = "ruff-fuzz"
+version = "0.0.0"
+authors = [
+  "Addison Crump <research@addisoncrump.info>",
+]
+publish = false
+edition = "2021"
+
+[features]
+full-idempotency = []
+
+[package.metadata]
+cargo-fuzz = true
+
+[dependencies]
+arbitrary = { version = "1.3.0", features = ["derive"] }
+libfuzzer-sys = { version = "0.4.1" }
+ruff = { path = "../crates/ruff" }
+ruff_python_ast = { path = "../crates/ruff_python_ast" }
+ruff_python_formatter = { path = "../crates/ruff_python_formatter" }
+rustpython-ast = { git = "https://github.com/astral-sh/RustPython-Parser.git", rev = "7a3eedbf6fb4ea7068a1bf7fe0e97e963ea95ffd", default-features = false, features = ["all-nodes-with-ranges"]}
+rustpython-parser = { git = "https://github.com/astral-sh/RustPython-Parser.git", rev = "7a3eedbf6fb4ea7068a1bf7fe0e97e963ea95ffd", default-features = false, features = ["full-lexer", "all-nodes-with-ranges"] }
+
+# Prevent this from interfering with workspaces
+[workspace]
+members = ["."]
+
+[[bin]]
+name = "ruff_parse_simple"
+path = "fuzz_targets/ruff_parse_simple.rs"
+
+[[bin]]
+name = "ruff_fix_validity"
+path = "fuzz_targets/ruff_fix_validity.rs"
+
+[[bin]]
+name = "ruff_parse_idempotency"
+path = "fuzz_targets/ruff_parse_idempotency.rs"
+
+[profile.release]
+opt-level = 3
+debug = true
+
+[profile.dev]
+opt-level = 3
+debug = true
```

Comment:
What's the motivation for opting in to opt-level 3 even for debug? Would a lower level like 1 or 2 suffice?

Reply:
Fuzzers rely on being able to execute quickly, and (if unsafe code is involved) some faults are not consistent. Having these be as close as possible is important. Comparatively, you can build the fuzzers on their own (just `cargo run ...` instead of `cargo fuzz run ...`) if you want to analyse a crash without [fuzzer instrumentation](https://clang.llvm.org/docs/SanitizerCoverage.html) (in some rare cases, the instrumentation can modify behaviour or optimisation). This is a rare use case but something I wanted to allow for.